src/foam/nanos/auth/UserAndGroupAuthService.js

```diff
@@ -478,5 +417,83 @@ foam.CLASS({
         return true;
       `
     }
+  ],
+  static: [
+    {
+      name: 'loginUser',
```

Comment:
There's probably a better place for this, a standalone local only service to properly sign a session into a user, an authentication service for signing in, and the existing authentication service for doing permission checks.

But this works for now